docs/sources/reference/api/docker_remote_api.md

```diff
@@ -66,6 +66,12 @@ Volumes are now initialized when the container is created.
 **New!**
 You can now copy data which is contained in a volume.
 
+`POST /build`
+
+**New!**
+This endpoint now optinally takes a serialized array of build-time environment
```

Comment:
Thanks for reviewing @EronWright. I have fixed the spelling and updated the diff